plutus-core/executables/src/PlutusCore/Executable/Parsers.hs

```diff
@@ -159,6 +160,32 @@ exampleSingle = ExampleSingle <$> exampleName
 exampleOpts :: Parser ExampleOptions
 exampleOpts = ExampleOptions <$> exampleMode
 
+builtinSemanticsVariantReader :: String -> Maybe (BuiltinSemanticsVariant DefaultFun)
+builtinSemanticsVariantReader =
+    \case
+     "1" -> Just DefaultFunSemanticsVariant1
+     "2" -> Just DefaultFunSemanticsVariant2
+     _   -> Nothing
+
+-- This is used to make the help message show you what you actually need to type.
+showBuiltinSemanticsVariant :: BuiltinSemanticsVariant DefaultFun -> String
```

Comment:
[There is one](https://github.com/input-output-hk/plutus/blob/3ff3901e7766647a135fa8442aaeec6d98a8b4c5/plutus-core/plutus-core/src/PlutusCore/Default/Builtins.hs#L1059), but it's the standard one, and if you use that it says that the default option is `DefaultFunSemanticsVariant2`.  I just want to say `1` or `2` instead of having to type all of that in, so I had to add a special function to get a sensible help message.